pkg/build/nodeimage/const.go

```diff
@@ -19,7 +19,8 @@ package nodeimage
 // these are well known paths within the node image
 const (
 	// TODO: refactor kubernetesVersionLocation to a common internal package
-	kubernetesVersionLocation      = "/kind/version"
-	defaultCNIManifestLocation     = "/kind/manifests/default-cni.yaml"
-	defaultStorageManifestLocation = "/kind/manifests/default-storage.yaml"
+	kubernetesVersionLocation            = "/kind/version"
+	defaultCNIManifestLocation           = "/kind/manifests/default-cni.yaml"
+	defaultStorageManifestLocation       = "/kind/manifests/default-storage.yaml"
+	defaultNetworkPolicyManifestLocation = "/kind/manifests/default-network-policy.yaml"
```

Comment:
Actually, I think maybe we should put a second daemonset in the *same* yaml file.

Then it will "just work" forwards/backwards with older kind versions / images

Reply:
(even if "CNI" is a poor name for this, it's commonly conflated anyhow)

Reply:
done